tests/e2e/lib/apps.go

```diff
@@ -45,17 +47,40 @@ var (
 	e2eAppLabelSelector       = labels.NewSelector().Add(*e2eAppLabelRequirement)
 )
 
-func InstallApplication(ocClient client.Client, file string) error {
-	return InstallApplicationWithRetries(ocClient, file, 3)
+func InstallApplication(ocClient client.Client, file string, replace ...string) error {
+	return InstallApplicationWithRetries(ocClient, file, 3, replace...)
 }
 
-func InstallApplicationWithRetries(ocClient client.Client, file string, retries int) error {
+func InstallApplicationWithRetries(ocClient client.Client, file string, retries int, replace ...string) error {
 	template, err := os.ReadFile(file)
 	if err != nil {
 		return err
 	}
 	obj := &unstructured.UnstructuredList{}
 
+	// YAML templates can have replace directives, in the form of "<<template-replace-N>>", where N
```

Comment:
Is this oadp specific directive or we're following some published standard?

Reply:
just from my head :rofl: 

tried to think what would be good in the future using this

I accept suggestions